packages/terra-site/webpack.config.js

```diff
@@ -23,6 +23,7 @@ const threadLoaderRule = {
 
 module.exports = {
   entry: {
+    raf: 'raf/polyfill',
```

Comment:
Why are we using two polyfills now? Will this be required by all consumers?

Reply:
React 16 requires request animation frame. https://reactjs.org/blog/2017/09/26/react-v16.0.html